BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -134,6 +134,10 @@
 
   BoardSupportLib|Platform/CommonBoardPkg/Library/BoardSupportLib/BoardSupportLib.inf
 
+!if $(ENABLE_CSME_UPDATE) == 0

```

Comment:
This is only for FirmwareUpdate payload. So, it needs to be handled in FirmwareUpdate.inf and it looks only CsmeUpdateDriver is using MeFwUpdateLib.
What about having a single CsmeUpdateLib which has both CsmeUpdateDriver and MeFwUpdateLib?

Reply:
Removed this